cx_freeze_prisma_reproduction/app.py

```diff
@@ -8,6 +10,8 @@ async def main():
     set_start_method("spawn", force=True)
     freeze_support()
 
+    if getattr(sys, "frozen", False):
+        os.environ["PRISMA_BINARY_CACHE_DIR"] = os.path.dirname(sys.executable)
```

Comment:
From what I read (https://github.com/prisma/prisma-engines?tab=readme-ov-file#building-prisma-engines) , I think we can use the variable PRISMA_BINARY_PATH w/ the full name, ie
 os.environ["PRISMA_BINARY_PATH"] = os.path.join(os.path.dirname(sys.executable), "query-engine-windows.exe")

and only include it in setup.
But there are others engines to copy and variables to set?